crates/lib/src/qpu/rpcq.rs

```diff
@@ -176,10 +176,22 @@ impl<'params, T: Serialize> RPCRequest<'params, T> {
             params,
             id: Uuid::new_v4().to_string(),
             jsonrpc: "2.0",
-            client_timeout: 10,
+            client_timeout: Some(30),
```

Comment:
I've seen this number used twice in code and once (with wrong value) in docs. Should make this a constant, maybe even public so the default value is visible in the public API.

Reply:
Fair enough. I created a constant for it - though I created a separate one for this RPCQ client and the quilc module because I don't think it makes sense for them to be coupled.